seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -759,18 +759,15 @@
-        "The format is similar to an external link except instead of a URL, you'll use the <code>#</code> and a word to describe the section you want to jump to.",
-        "Here's an example:",
-        "<code>&lt;a href=\"#contact\"&gt;Go to contact section&lt;/a&gt;</code>",
-        "Next, you'll want to create a corresponding anchor link in the HTML where you want the internal link to take users. Instead of using the <code>href</code> attribute, you'll use the <code>name</code> attribute. Here's an example:",
-        "<blockquote>&lt;h1&gt;Contact&lt;/h1&gt;<br>&lt;a name=\"contact\"&gt;&lt;/a&gt;</blockquote>",
-        "Now when users click the \"Go to contact section\" link, they'll be taken to the section of the webpage with the anchor that has the <code>name</code> attribute \"contact\".",
+        "Anchor or <code>a</code> elements can also be used to create internal links to jump to different sections within a webpage.",
+        "To create an internal link, you assign a link's <code>href</code> attribute to a hash symbol <code>#</code> plus the <code>id</code> attribute for an element that you want to internally link to, usually further down the page. You would then need to add the same <code>id</code> attribute to the element you are linking to. An <code>id</code> is an attribute that uniquely describes an element.",
+        "Below is an example of an internal anchor link and its target element:",
+        "<blockquote>&lt;a href=\"#contactsHeader\"&gt;Contacts&lt;/a&gt;<br>...<br>&lt;h2 id=\"contactsHeader\"&gt;Contacts&lt;/h2&gt;</blockquote>",
+        "When users click the <u>Contacts</u> link, they'll be taken to the section of the webpage with the <b>Contacts</b> header element.",
         "<hr>",
-        "Change your external link to an internal link by changing the <code>href</code> attribute to \"#bottom\" and the anchor text from \"cat photos\" to \"Jump to Bottom\".",
-        "Don't forget to remove the <code>target=\"_blank\"</code> attribute since this causes the linked document to open in a new window tab.",
-        "Then add an anchor link with a <code>name</code> attribute set to \"bottom\" after the last paragraph.",
-        "<strong>Note</strong><br>Additional kitty ipsum text has been added to the paragraphs to better show the effect of clicking the internal anchor link to jump to the bottom of the page. Also, this change will only apply to this challenge. The next challenge goes back to showing your external link."
+        "Change your external link to an internal link by changing the <code>href</code> attribute to \"#footer\" and the text from \"cat photos\" to \"Jump to Bottom\".",
+        "Remove the <code>target=\"_blank\"</code> attribute from the anchor tag since this causes the linked document to open in a new window tab.",
+        "Then add a <code>footer</code> element at the bottom of the page after the <code>main</code> element. Give it an <code>id</code> attribute of \"footer\"."
```

Comment:
This is the first time in the curriculum a camper is asked to use the `footer` element. It's only mentioned once before. I'm concerned that this might make this challenge more complicated than it needs to be. Maybe we should make them jump to the last paragraph? Or we add a paragraph at the bottom that says "bottom", which they then add the id to?